sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -1040,6 +1047,13 @@ public void deletePermanent(Case caze) {
 		super.deletePermanent(caze);
 	}
 
+	private void visitSubquery(CriteriaBuilder cb, Root<Visit> visitRoot, Subquery<Long> contactVisitsSubquery) {
```

Comment:
```suggestion
	private void initVisitSubqueryForDeletion(CriteriaBuilder cb, Root<Visit> visitRoot, Subquery<Long> contactVisitsSubquery) {
```

The new name was a bit too generic now :D sorry for not being clear enough